.github/workflows/release.yaml

```diff
@@ -0,0 +1,98 @@
+name: Release
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  versions:
+    runs-on: ubuntu-latest
+    outputs:
+      python_version: ${{ steps.versions.outputs.python_version }}
+      release_version: ${{ steps.versions.outputs.release_version }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get latest supported Python version
+        id: versions
+        run: |
+          CLASSIFIERS="$(yq -p toml -o toml '.project.classifiers[]' pyproject.toml | grep "Programming Language :: Python :: " | grep "\.")"
+          VERSION="$(echo "${CLASSIFIERS}" | sed -e 's/Programming Language :: Python :: \([0-9.]*\)$/"\1"/g' | tail -n 1)"
+          echo "python_version=[${VERSION}]" > "${GITHUB_OUTPUT}"
+      # See: https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions
+      - name: Get release version
+        run: echo "release_version=$(GITHUB_REF#refs/*/}" >> "${GITHUB_OUTPUT}"
```

Comment:
I could also grab this from `pyproject.toml` - it's a matter of what I want to be the source of truth.